src/app/pages/component-viewer/component-viewer.html

```diff
@@ -1,22 +1,24 @@
 <!-- TODO(jelbourn): turn this into nav tabs -->
-<md-tab-group class="docs-component-viewer-tabbed-content"
+<md-tab-group role="tablist" class="docs-component-viewer-tabbed-content"
```

Comment:
The roles and other aria attributes are something that really belong as part of the md-tab-group implementation